path_access_token.go

```diff
@@ -70,6 +70,16 @@ func (b *azureSecretBackend) secretAccessTokenResponse(ctx context.Context, stor
 		return nil, err
 	}
 
+	if b.Logger().IsDebug() {
+		nacl, err := b.Salt(ctx)
+		if err != nil {
+			b.Logger().Error("error creating access token", "error", err)
+		}
+		hmac := nacl.GetIdentifiedHMAC(role.Credentials.Password)
```

Comment:
this ends up being the same salting used in the audit endpoints?

Reply:
In practice, I will not need to use `/sys/audit-hash/:path` to compare hashes. I'm interested only in whether or not the password/credential of a given application is changing.

Reply:
is this temporary or intended for use long term?

Reply:
I don't think we should keep this long term but I do think it could stick around for a couple months since this updated plugin likely will have to be deployed to every vault cluster and then configured in vault-config.

This is because i'll need to refresh the Azure credentials for each datacenter and then wait until this problem rears its head again.